odoo/client.go

```diff
@@ -43,15 +66,20 @@ type loginParams struct {
 //  - the credentials were wrong,
 //  - encoding or sending the request,
 //  - or decoding the request failed.
-func (c Client) Login(ctx context.Context, login, password string) (*Session, error) {
-	resp, err := c.requestSession(ctx, login, password)
+func (c Client) Login(ctx context.Context) (*Session, error) {
```

Comment:
One idea: We could merge `NewClient` and `Login` (and call it `Dial` or `Open`?). We will pretty much always call them in order anyway now as we provide the username and password as part of `NewClient`

Reply:
I like!